client/src/shared/layouts/Layout-learning/Layout-learning.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import NextIcon from '../../icons/next';
+import PreviousIcon from '../../icons/previous';
+import SideBar from './SideBar';
+
```

Comment:
* incorrect directory
* use lowercase for the directory.
instead of `client/src/shared/layouts/Layout-learning/Layout-learning.tsx`
it should be `client/src/shared/layouts/layout-learning.tsx`